src/styles/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
U can move font-family and main color to the html {}

Reply:
Agree, set font styles to `html` tag